frame/header-mmr/src/lib.rs

```diff
@@ -105,10 +105,38 @@ pub mod pallet {
 	#[pallet::getter(fn peak_of)]
 	pub type Peaks<T: Config> = StorageMap<_, Identity, NodeIndex, T::Hash, OptionQuery>;
 
+	/// Migration step.
+	#[pallet::storage]
+	#[pallet::getter(fn migration_step)]
+	pub type MigrationStep<T> = StorageValue<_, u32, ValueQuery, DefaultMigrationStep>;
+	#[pallet::type_value]
+	pub fn DefaultMigrationStep() -> u32 {
+		500
+	}
+
```

Comment:
LGTM, hard code using 500 is simpler?

Reply:
I'm not sure if this will affect the chain performance.

Reply:
make sense. We did not have clean the storage right, if we do not set any value.

Reply:
Yes.

Reply:
I have another about this piece of code. How would you configure it later without a runtime upgrade? I didn't see any setter function. If we can't modify it via signed transaction, has no difference with hard code 500.  @AurevoirXavier

Reply:
Sudo/Gov